tests/matchers/test_compute_overlap.py

```diff
@@ -19,19 +19,22 @@ def test_get_labels_with_overlap(overlap):
         img_size=256, num_labels=0, sequential=True, seed=1
     )
 
-    perfect_gt, perfect_res, perfect_ious = get_labels_with_overlap(
-        image1, image1, overlap
-    )
-    assert list(perfect_gt) == list(range(1, n_labels + 1))
-    assert list(perfect_res) == list(range(1, n_labels + 1))
-    assert list(perfect_ious) == [1.0] * n_labels
+    ious = get_labels_with_overlap(image1, image1, overlap)
+    assert [gt for gt, _, _ in ious] == list(range(1, n_labels + 1))
```

Comment:
here I think you could avoid the triple iteration by doing

```python
gt, res, iou = tuple(zip(*ious))
assert list(gt) == list(range(1, n_labels+1))
...
```

Up to you if the change is worth it